torch/jit/__init__.py

```diff
@@ -717,7 +717,8 @@ class _ConstSequential(_ConstModuleList):
     def __init__(self, mods):
         super(_ConstSequential, self).__init__(mods._modules.values())
 
-    @script_method
+    # commenting script_method until T29956663 is resolved
+    #@script_method
```

Comment:
Why did this change? Can you open a public issue describing the problem?

Reply:
this change can be removed. I'll push a patch shortly to address this. We need to wrap ConstSequential as a string source, to get around opt mode only shipping `.pyc` files (and no .py files).

This change as-is will also fail oss contbuilds